### PR TITLE
Cleanup the approvals section on the course run detail page

### PR DESCRIPTION
**Pull request by asadazam93**
ECOM-7731

On the Course-run detail page, under the Approvals section, the role and names are now listed in the following order:
Course Team, Project Coordinator, Course Preview, Publisher.
Also removed Marketing from this list.

### CODE REVIEW

File: course_discovery/apps/publisher/views.py [hunk 1]
```diff
@@ -681,11 +681,11 @@ def get_context_data(self, **kwargs):
 def get_course_role_widgets_data(user, course, state_object, change_state_url, parent_course=False):
     """ Create role widgets list for course user roles. """
     role_widgets = []
-    course_roles = course.course_user_roles
+    course_roles = course.course_user_roles.exclude(role__in = [PublisherUserRole.MarketingReviewer])
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/views.py:684]: `course_roles = course.course_user_roles.exclude(role=PublisherUserRole.MarketingReviewer)`

**Review by waheedahmed — CHANGES_REQUESTED** [review]
One comment and also fix translations.

File: course_discovery/apps/publisher/views.py [hunk 2]
```diff
@@ -681,11 +681,11 @@ def get_context_data(self, **kwargs):
...
     roles = [PublisherUserRole.CourseTeam, PublisherUserRole.ProjectCoordinator]
     if parent_course:
         roles = [PublisherUserRole.CourseTeam, PublisherUserRole.MarketingReviewer]
-        course_roles = course_roles.filter(role__in=roles)
+        course_roles = course.course_user_roles.filter(role__in=roles)
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/views.py:688]: Why this change ? You have already doing query `course.course_user_roles` on [line 684](https://github.com/edx/course-discovery/pull/846/files#diff-2680d228b92a6ff1a837bbd2abcd1cfbR684)

Comment (waheedahmed) [reply course_discovery/apps/publisher/views.py:688]: @tasawernawaz Now we need different roles for both course detail and course-run detail pages thats he is excluding MKTG role from roles for course-run.

**Comment by asadazam93** [comment]
I have created a new for this: https://github.com/edx/course-discovery/pull/849
